magento/adobe-stock-integration
# #1476 :- Generate renditions for all images uploaded to magento

### Description (*)

This PR will generate renditions image when image is uploaded using Magento Enhanced Media Gallery Image Uploader
### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1476: Generate renditions for all images uploaded to magento

### Manual testing scenarios (*)

1.  Configure width/height field from system configuration fields ()
2. Upload image using Enhanced media gallery
3. Check pub/media directory to see the `.renditions` images

### Expected Result
![image](https://user-images.githubusercontent.com/39480008/85099089-18038880-b21a-11ea-9946-a78cf1356c6a.png)

![image](https://user-images.githubusercontent.com/39480008/85104130-a2051e80-b225-11ea-810f-03086ab0fd65.png)
